tests/frameworks/test_pyramid.py

```diff
@@ -245,3 +245,72 @@ def test_exception(self):
         self.assertIsNotNone(urllib3_span.stack)
         self.assertTrue(type(urllib3_span.stack) is list)
         self.assertTrue(len(urllib3_span.stack) > 1)
+
+    def test_response_header_capture(self):
+        from instana.singletons import agent
```

Comment:
I would delete this line and change line 11 to `from instana.singletons import agent, tracer`